bootstrap/services/storm-bundle.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "STORM",
+  "registerClass": "com.hortonworks.streamline.streams.cluster.register.impl.StormServiceRegisterer",
+  "serviceUISpecification": {
+    "fields": [
+      {
+        "uiName": "UI server hostname",
+        "fieldName": "uiServerHostname",
+        "isOptional": false,
+        "tooltip": "Hostname of Storm UI server",
+        "type": "string"
+      },
+      {
+        "uiName": "Nimbuses hostnames",
+        "fieldName": "nimbusesHostnames",
+        "isOptional": false,
+        "tooltip": "Hostnames of Nimbuses",
+        "type": "array.string"
+      },
+      {
+        "uiName": "storm.yaml",
+        "fieldName": "storm.yaml",
+        "isOptional": false,
```

Comment:
Why is this mandatory ? If users dont want to override cant they run with defaults? 

Also I am  not clear on how and where this storm.yaml file would be used.